openapi/p4pa-process-executions.openapi.yaml

```diff
@@ -1,31 +1,98 @@
-openapi: 3.1.0
+openapi: 3.0.1
 info:
   title: p4pa-process-executions
   description: Api and Models
-  version: "1.0.0"
+  version: 0.0.1
+servers:
+  - url: http://localhost:8080
+    description: Generated server url
 paths:
-  /api/v1/greet:
-    get:
-      summary: "Hello World"
-      description: "Sample endpoint that return greetings"
+  /:
+    post:
+      summary: create-ingestion-flow-file
+      tags:
+        - ingestion-flow-file-controller
+      operationId: createIngestionFlowFile
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/IngestionFlowFileDTO'
+        required: true
       responses:
         '200':
-          description: "Success"
+          description: OK
           content:
             application/json:
               schema:
-                type: object
-                properties:
-                  message:
-                    type: string
-                    example: "Hello, World!"
-        '500':
-          description: "Internal Server Error"
-          content:
-            application/json:
-              schema:
-                type: object
-                properties:
-                  error:
-                    type: string
-                    example: "Internal Server Error"
+                $ref: '#/components/schemas/IngestionFlowFileDTO'
+components:
+  schemas:
+    IngestionFlowFileDTO:
+      type: object
+      required:
+        - organizationId
+        - status
+        - flowIdentifierCode
+        - flagActive
+        - operatorExternalUserId
```

Comment:
this is not required, it should be set by the this microservice (see message I left on Slack)